.travis.yml

```diff
@@ -0,0 +1,51 @@
+sudo: required
+
+language: node_js
+
+node_js:
+  - "10.3.3"
+
+git:
+  depth: 2
+
+services:
+  - docker
+
+env:
+  global:
+    - IMAGE_REPO=gcr.io/dd-decaf-cfbf6/caffeine
+    - IMAGE_TAG=travis-ci-test
+    - IMAGE=${IMAGE_REPO}:${IMAGE_TAG}
+
+before_script:
+  - npm install -g @angular/cli
+
+install:
+  - npm install
```

Comment:
did you make some research if it is possible to use `npm ci`? That would significantly bring down the build time

Reply:
I didnt, but I did right now, and we can use, and it is better.